Sources/Quick/DSL/DSLValidation.swift

```diff
@@ -0,0 +1,28 @@
+/**
+    Descriptors intended to identify DSL function context.
+*/
+public enum ExampleContextDescriptor: String {
```

Comment:
Thanks for pointing this out. I think this was simply an oversight on my part. It should absolutely be limited to internal access only. I've made that change in my in-flight update.